karma.conf.js

```diff
@@ -71,11 +71,12 @@ module.exports = function (config) {
             // This package doesn't work, specifically the reference to `vector_tile.js`, it needs
             // to be fixed, something like the following should work... but doesn't and needs to be
             // investigated.
-            // {
-            //     pattern: "@here/harp-vectortile-datasource/lib/adapters/omv/proto/vector_tile.js",
-            //     included: true
-            // },
-            //"@here/harp-vectortile-datasource/**/*.ts"
+            {
+                pattern:
+                    "@here/harp-vectortile-datasource/lib/adapters/omv/proto/vector_tile.js",
+                included: false
+            },
+            "@here/harp-vectortile-datasource/**/*.ts",
```

Comment:
This is adding all files / tests from this package to the bundle that karma typescript creates.

The file above `vector_tile.js` is added from `OmvData.ts`